src/test/java/uk/co/ramp/event/InfectionEventProcessorTest.java

```diff
@@ -38,23 +24,30 @@
 import uk.co.ramp.people.Case;
 import uk.co.ramp.statistics.StatisticsRecorder;
 
-@RunWith(SpringRunner.class)
-@DirtiesContext
-@Import({TestUtils.class, AppConfig.class, TestConfig.class})
+//
+// @RunWith(SpringRunner.class)
+// @DirtiesContext
+// @Import({TestUtils.class, AppConfig.class, TestConfig.class})
 public class InfectionEventProcessorTest {
   private InfectionEventProcessor eventProcessor;
   private DiseaseProperties diseaseProperties;
 
-  @Autowired private StandardProperties properties;
-  @Autowired private DistributionSampler distributionSampler;
-  @Autowired private StatisticsRecorder statisticsRecorder;
+  private StandardProperties properties;
+  private DistributionSampler distributionSampler;
+  private StatisticsRecorder statisticsRecorder;
 
   private Case thisCase;
-
   private InfectionEvent event;
 
   @Before
   public void setUp() throws FileNotFoundException {
+
+    properties = mock(StandardProperties.class);
+    when(properties.timeStepsPerDay()).thenReturn(1);
+    distributionSampler = mock(DistributionSampler.class);
+    when(distributionSampler.getDistributionValue(any())).thenCallRealMethod();
```

Comment:
Ideally we should mock the resulting behaviour here as well, but as it was being auto wired before, this change is definitely one step in the right direction. We can address this in another PR if/when needed

Reply:
Something like this? 
```suggestion
        when(distributionSampler.getDistributionValue(any())).thenAnswer(i -> ((int) Math.round(((Distribution) i.getArgument(0)).mean())));

```

Reply:
@sam-brett Yes, that looks good to me. I think there are two other places in this PR where the same change can be made